web-console/knikubevirt/wizard-import/wizard-import.md

```diff
@@ -0,0 +1,73 @@
+# Create A Virtual Machine
+
+## Import VM
+
+The ‘Import virtual machine’ action will be available in the ‘Create virtual machine’ dropdown.
+
+![Empty state](img/empty-state.png)
+
+![wizard default screen](img/default.png)
+
+### Selecting provider
+
+The Provider is chosen (in this case VMware is the only one available).
+
+![selecting provider](img/selecting-provider.png)
+
+### Selecting vCenter instance
+
+Once VMware is selected a new vCenter instance field appears. The user clicks it to see a list of previously-connected vCenter instances that have been stored as Kubernetes Secrets. The user selects one.
+
+![selecting vcenter instance](img/selecting-vcenter-instance.png)
+
+### Connecting to new instance
+
+The user can also connect to a new vCenter instance by selecting ‘Connect to new instance’ from within the dropdown.
+
+![connecting to new instance](img/connecting-to-new-instance.png)
+
+New fields for vCenter URL, vCenter Username, vCenter Password, and VM to import appear along with a checkbox to ‘Save as new vCenter instance secret’. This would be creating a new secret so it would be unchecked by default.
+The vCenter URL field should include a syntax hint with the desired formatting. The user is likely to paste a variety of URLs, so any FQDN the user submits should be validated and automatically corrected before being used to communicate with vCenter’s API.
+
+![new fields required](img/new-fields-required.png)
+
+When the user clicks the “Check” button a spinner icon appears within the button to indicate that the credentials are being checked.
+
+![credentials checked](img/credentials-checked.png)
+
+### Selecting virtual machine
+
+When a vCenter instance is selected, The ‘Virtual Machine to import’ dropdown is available with all the available VMs.
+
+![selecting virtual machine](img/selecting-virtual-machine.png)
+
+### vCenter connection error
+
+If the wizard cannot connect to the vCenter instance using the credentials stored within its Secret, an error message will be displayed. If the credentials are incorrect they can update them. If they check ‘update vCenter credentials secret’ the action will be ‘check and update’. This should be the default because they have already created this secret. This allows them to update the secret without having to leave this flow.
+
+![vcenter connection error](img/vcenter-connection-error.png)
+
+![vcenter connection incorrect credentials](img/vcenter-connection-incorrect-credentials.png)
+
+Once the credentials are checked and updated the username/password fields collapse.
```

Comment:
It's not included in the initial designs here but I wonder if upon successfully updating the credentials and having the username/password field collapse we should have "Credentials successfully updated" text under the vCenter instance dropdown.